### ISSUE TITLE
[cetic/nifi] 0.6.1 Server with no auth skips setting custom nifi.properties

### ISSUE THREAD

**Issue by jdesroch** [cetic/helm-nifi#127]
**Describe the bug**
nifi.properties values defined in `.Values.properties.safetyValve` are ignored unless `.Values.auth.oidc.enabled`

**What happened**:
0.6.1 adds the check `{{- if .Values.auth.oidc.enabled }}` on line 193 of templates/statefulset.yaml. This check means that custom nifi.properties values (line 215) are never set if .Values.auth.oidc is disabled.

**What you expected to happen**:
- Remove/refactor check for OIDC auth
- Modify nifi.properties using safetyValve values regardless of auth or other settings. 

**How to reproduce it** (as minimally and precisely as possible):
Vanilla deploy of 0.6.1 with 2 replicas and no auth.
Add values to verify change
```
.Values.properties.safetyValve:
    nifi.ui.banner.text: 'Show this text'
```
Look for banner text in UI.

**Anything else we need to know**:
Related to #115 & #108

**Comment by AyadiAmen** [comment]
https://github.com/cetic/helm-nifi/pull/126

**Comment by ronny-panknin-flaconi** [comment]
please reopen the issue, this is not fixed with #126
